apps.yml

```diff
@@ -3755,3 +3755,14 @@ apps:
     name: fxa-support-panel.prod.mozaws.net
     op: auth0
     url: https://fxa-support-panel.prod.mozaws.net/openid/callback/login
+- application:
+    authorized_groups:
+    - team_netops
+    - team_infra
+    authorized_users: []
+    client_id: OSKMKPkvUoDg0K8agd4iHSwUam0lpr6p
+    display: false
+    logo: auth0.png
+    name: Console View
```

Comment:
```suggestion
    name: ConsoleView
```

Reply:
I'm switching it from View to Flow, since that's the domain

Reply:
Oh right, good catch.